.github/workflows/cypress_test.yml

```diff
@@ -0,0 +1,15 @@
+name: Cypress Tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    name: Headless Tests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      # Install npm dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
+        with:
+          command: npm run cypress-headless
```

Comment:
can you execute tests in chrome headless so that the pipelines are comparable?
(Playwright tests run in chrome in Github workflow)